### PR TITLE
Newsletters: add 243 (2023-03-22)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, client/services, topic links (and maybe some news, if any happens) @harding 
- [x] Client/services @bitschmidty 
- [x] LND 7448 @Xekyo (this is not super notable, but I think it shows that rebroadcasting is something every individual wallet is solving for, so maybe more effort should be put into global solutions (like node-level rebroadcasting) or reusable solutions (e.g. bdk-like libraries for wallets))

### CODE REVIEW

File: _posts/en/newsletters/2023-03-22-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  unconfirmed transactions especially to address transactions that were
+  evicted from mempools. When enabled, the rebroadcaster will submit
+  unconfirmed transactions to the attached full node once per block
+  until it is confirmed. LND was already rebroadcasting transactions in
+  a similar fashion when operating in Neutrino-mode. As noted in a
+  previously covered the Stack Exchange Q&A, [Bitcoin Core currently
+  does not rebroadcast transactions][no rebroadcast], although it would
+  be desirable for privacy and reliability if full node behavior were
+  amended to rebroadcast any transactions that the node had expected to
+  have been included in the prior block. Until then, it is the wallets’
```
Comment (bitschmidty) [inline _posts/en/newsletters/2023-03-22-newsletter.md:90]: maybe

```suggestion
  have been included in the prior block. Until then, it is the wallet’s
```

File: _posts/en/newsletters/2023-03-22-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  support][mempool tweet] for [CPFP][topic cpfp]-related transactions.
+
+- **Sparrow v1.7.3 released:**
+  Sparrow's [v1.7.3 release][sparrow v1.7.3] includes [BIP129][] support for multisig wallets (see
+  [Newsletter #136][news136 bsms]) and custom block explorer support among other features.
+
+- **Stack Wallet adds coin control, BIP47:**
+  Recent releases of the [Stack Wallet][github stack wallet] add [coin
+  control][topic coin selection] features and [BIP47][] support.
+
```
[the remark this answers is not in the document]
Comment (bitschmidty) [reply _posts/en/newsletters/2023-03-22-newsletter.md:59]: @MarnixCroes thanks for the heads up, looks like this was released an hour ago! 

I made a small change to lowercase taproot (see Optech [style guide relevant section](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/STYLE.md#common-nouns))

File: _posts/en/newsletters/2023-03-22-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+- **Xapo Bank supports Lightning:**
+  Xapo Bank [announced][xapo lightning blog] its customers can now send outgoing
+  Lightning payments from the Xapo Bank mobile apps, using underlying infrastructure from Lightspark.
+
+- **TypeScript library for miniscript descriptors released:**
+  The TypeScript-based [Bitcoin Descriptors Library][github descriptors library]
+  has support for [PSBTs][topic psbt], [descriptors][topic descriptors],
+  [miniscript][topic miniscript], and signing support including some hardware
+  signing devices.
```
Comment (Xekyo) [inline _posts/en/newsletters/2023-03-22-newsletter.md:30]: This feels a bit unclear. The enumeration describes a few different technologies, but the signing support seems to refer to all of the former as well as the following hardware signing devices. Perhaps splitting the sentence would clarify:

```suggestion
  The TypeScript-based [Bitcoin Descriptors Library][github descriptors library]
  has support for [PSBTs][topic psbt], [descriptors][topic descriptors],
  and [miniscript][topic miniscript]. This includes support for signing directly, or via some hardware signing devices.
```

File: _posts/en/newsletters/2023-03-22-newsletter.md (hunk 1 above)
Comment (Xekyo) [reply _posts/en/newsletters/2023-03-22-newsletter.md:92]: That works as well. :+1:

File: _posts/en/newsletters/2023-03-22-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  be desirable for privacy and reliability if full node behavior were
+  amended to rebroadcast any transactions that the node had expected to
+  have been included in the prior block. Until then, it is the wallets’
+  responsibility to ensure the presence of transactions of interest in
+  mempools.
+
+- [BDK #793][] is a major restructuring of the library based on the work
+  of the [bdk_core sub-project][].  According to the PR description, it
+  "maintains the existing wallet API as much as possible and adds very
+  little."  Three seemingly minor changed APIs are listed in the PR
```
Comment (Xekyo) [inline _posts/en/newsletters/2023-03-22-newsletter.md:99]: How about:

```suggestion
  little."  Three API endpoints with seemingly minor changes are listed in the PR
```

File: _posts/en/newsletters/2023-03-22-newsletter.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+- **PSBT-based exchange OpenOrdex launches:**
+  The [open source][github openordex] exchange software allows sellers to create
+  an order book of Ordinal satoshis using [PSBTs][topic psbt] and buyers to sign
+  and broadcast to complete the trade.
+
+- **BTCPay Server coinjoin plugin released:**
+  The Wasabi Wallet [announcement][wasabi blog] notes that any BTCPay Server
+  merchant can activate the optional plugin which supports the
+  [WabiSabi][news194 wabisabi] protocol for [coinjoins][topic coinjoin].
```
Comment (harding) [inline _posts/en/newsletters/2023-03-22-newsletter.md]: Changing this internal link to newsletter 101, which I think has a better description (and no distraction about payjoins).

Comment (bitschmidty) [reply _posts/en/newsletters/2023-03-22-newsletter.md]: ACK on this and the other client/services section updates.

**Pull request merged**